lib/web_ui/test/engine/platform_views/message_handler_test.dart

```diff
@@ -89,27 +90,37 @@ void testMain() {
                   'The response should be a success envelope, with null in it.');
         });
 
-        test('calls a contentHandler with the result of creating a view',
+        test('inserts the created view into the platformViewsContainer',
             () async {
+          final DomElement platformViewsContainer = createDomElement('pv-container');
           contentManager.registerFactory(
               viewType, (int id) => createDomHTMLDivElement()..id = 'success');
           final PlatformViewMessageHandler messageHandler = PlatformViewMessageHandler(
+            platformViewsContainer: platformViewsContainer,
             contentManager: contentManager,
-            contentHandler: contentCompleter.complete,
           );
           final ByteData? message = _getCreateMessage(viewType, viewId);
 
           messageHandler.handlePlatformViewCall(message, completer.complete);
 
-          final DomElement contents = await contentCompleter.future;
           final ByteData? response = await completer.future;
 
-          expect(contents.querySelector('div#success'), isNotNull,
-              reason:
-                  'The element created by the factory should be present in the created view.');
-          expect(codec.decodeEnvelope(response!), isNull,
-              reason:
-                  'The response should be a success envelope, with null in it.');
+          expect(
+            platformViewsContainer.children.single,
+            isNotNull,
+            reason: 'The container has a single child, the created view.',
+          );
+          final DomElement platformView = platformViewsContainer.lastElementChild!;
```

Comment:
No need to add `lastElementChild` for this test, right? can this be:

```suggestion
          final DomElement platformView = platformViewsContainer.children.single!;
```

?